meetings/cranelift/2021/cranelift-11-29.md

```diff
@@ -12,7 +12,7 @@
     1. _Sumbit a PR to add your announcement here_
 1. Other agenda items
     1. _Sumbit a PR to add your item here_
-
+    Bug scrub - Maybe this belongs as a Wasmtime open but there are 369 Wasmtime issues dating back to 2016. Surely many are cranelift specific. Many are bugs likely in code that no longer exists. What's the best way to scrub these?
```

Comment:
:+1: There are 9 search results for peepmatic: https://github.com/bytecodealliance/wasmtime/issues?q=is%3Aissue+is%3Aopen+peepmatic I think these can be closed too. There are also several wasi issues that I think should be moved to https://github.com/WebAssembly/WASI.